bootstrap/Glue/bootstrap.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use Spryker\Glue\GlueApplication\Bootstrap\AbstractGlueBootstrap;
+use Spryker\Shared\Config\Application\Environment;
+use Spryker\Shared\ErrorHandler\ErrorHandlerEnvironment;
+
+define('APPLICATION', 'GLUE');
+defined('APPLICATION_ROOT_DIR') || define('APPLICATION_ROOT_DIR', realpath(__DIR__ . '/../../../../../..'));
+
+require_once APPLICATION_ROOT_DIR . '/vendor/autoload.php';
+
+Environment::initialize();
+
+$errorHandlerEnvironment = new ErrorHandlerEnvironment();
+$errorHandlerEnvironment->initialize();
+
+$bootstrap = new class extends AbstractGlueBootstrap
```

Comment:
Wouldn't it make sense to create a dedicated class for this? We should avoid using anonymous classes,